jetbrains-core/tst/software/aws/toolkits/jetbrains/services/sqs/SubscribeSnsDialogTest.kt

```diff
@@ -0,0 +1,85 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Also add a new test class and test for the action that makes sure it won't show up on FIFO?

Reply:
Added!